velox/vector/ComplexVector.h

```diff
@@ -722,6 +722,39 @@ class MapVector : public ArrayVectorBase {
   std::shared_ptr<MapVector> update(
       const folly::Range<DecodedVector*>& others) const;
 
+  /// Retrieves the specified keys and their associated values from each row in
+  /// a MapVector, returning a RowVector where each field corresponds to a key
+  /// and contains its respective value.
+  ///
+  /// For each row in the MapVector, the function extracts the values associated
+  /// with the specified keys and assigns them to the corresponding fields in
+  /// the output RowVector. If the map is null, a key is missing, or the value
```

Comment:
> If the map is null

I'd expect the output struct to be null, not have all null fields. Is there a reason for the currently-implemented semantic?